### ISSUE TITLE
can't run WebUI TypeError: 'numpy._DTypeMeta' object is not subscriptable

### ISSUE THREAD

**Issue by AlekFy** [Winfredy/SadTalker#716]
I tried git clone, than the zip release,
I've tried reinstalling and changing versions of OpenCV and Numpy with no luck
I'm using the venv provided
Each time I run webui.bat I get:

venv folder already exists, skipping creation...
venv "venv\Scripts\Python.exe"
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Commit hash: <none>
Installing requirements for SadTalker WebUI (may take longer time in first time)
Launching SadTalker Web UI
Traceback (most recent call last):
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\Launcher.py", line 199, in <module>
    start()
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\Launcher.py", line 192, in start
    from app import sadtalker_demo
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\app.py", line 3, in <module>
    from src.gradio_demo import SadTalker
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\src\gradio_demo.py", line 3, in <module>
    from src.utils.preprocess import CropAndExtract
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\src\utils\preprocess.py", line 2, in <module>
    import cv2, os, sys, torch
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\venv\lib\site-packages\cv2\__init__.py", line 181, in <module>
    bootstrap()
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\venv\lib\site-packages\cv2\__init__.py", line 175, in bootstrap
    if __load_extra_py_code_for_module("cv2", submodule, DEBUG):
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\venv\lib\site-packages\cv2\__init__.py", line 28, in __load_extra_py_code_for_module
    py_module = importlib.import_module(module_name)
  File "C:\Users\alekf\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "D:\StableDiffusion\SadTalker-0.0.2-rc\venv\lib\site-packages\cv2\typing\__init__.py", line 69, in <module>
    NumPyArrayGeneric = numpy.ndarray[typing.Any, numpy.dtype[numpy.generic]]
TypeError: 'numpy._DTypeMeta' object is not subscriptable

Launch unsuccessful. Exiting.

**Comment by toheedakhtar** [comment]
facing the same problem, did you found a solution.

**Comment by shareholdr** [comment]
Do this instead of hussle with the local version, full guide on how to get the Google colab variant to work here https://vm.tiktok.com/ZGeLNu2yq/

**Comment by decimecano** [comment]
> Do this instead of hussle with the local version, full guide on how to get the Google colab variant to work here https://vm.tiktok.com/ZGeLNu2yq/

I can barely see what's happening on the screen and the video doesn't have audio despite the fact you are talking. Can you summarize your method?
